docs/zz_generated.seed.ce.yaml

```diff
@@ -114,6 +114,9 @@ spec:
           network: ""
         # Kubevirt configures a KubeVirt datacenter.
         kubevirt:
+          # AdditionalNetPols (optional) allows to add some extra custom NetworkPolicies, that are deployed
+          # in the dedicated infra KubeVirt cluster. They are added to the default `cluster-isolation` one.
+          additionalNetPols: null
```

Comment:
could we rename to `customNetworkPolicies` and remove from description specific default network policies name? So that would be a change
> They are added to the default `cluster-isolation` one.

to 

> They are added to the defaults.

Reply:
done